lib/mix/tasks/generate_flow_charts.ex

```diff
@@ -15,47 +15,60 @@ defmodule Mix.Tasks.Ash.GenerateFlowCharts do
 
     * `--only` - only generates the given Flow file
 
+    * `--format` - Can be set to one of either:
+      * `plain` - Prints just the mermaid output as text. This is the default.
+      * `md` - Prints the mermaid diagram in a markdown code block.
+      * `svg` - Generates an SVG
+      * `pdf` - Generates a PDF
+      * `png` - Generates a PNG
+
   """
   use Mix.Task
 
   @shortdoc "Generates Mermaid Flow Charts for each `Ash.Flow`"
   def run(argv) do
     Mix.Task.run("compile")
 
-    {opts, _} = OptionParser.parse!(argv, strict: [only: :keep], aliases: [o: :only])
+    {opts, _} =
+      OptionParser.parse!(argv, strict: [only: :keep, format: :string], aliases: [o: :only])
```

Comment:
Should we add `-f` as an alias for `--format`?

Reply:
:+1: Seems consistent